### PR TITLE
add doc for local dev setup for legend studio/query with depot/sdlc

### PR DESCRIPTION
**Pull request by MauricioUyaguari**

### CODE REVIEW

File: docs/local-setups/sample-legend-depot-store-authorisedIdentities.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "Artifacts": [
+      "MauricioUyaguari"
```
Comment (akphi) [inline docs/local-setups/sample-legend-depot-store-authorisedIdentities.json:3]: really?

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**